src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,60 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import duke.taskList.TaskList;
+
+
+/**
+ * deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+
+    /**
+     * The constructor of the Storage class.
+     */
+    public Storage() {
+
+    }
+
+    /**
+     * Load the data from duke.txt into the TaskList inside of the program.
+     * @param list the TaskList that the data is loaded to from duke.txt.
+     */
+    public void load(TaskList list) {
```

Comment:
Can afford to be more specific here by specifying what you will be loading (Eg. loadTaskList)